packages/shared-ux/page/solution_nav/src/collapse_button.scss

```diff
@@ -26,12 +26,12 @@
   }
 
   &:not(&-isCollapsed) {
-    background-color: $euiColorEmptyShade !important; // Override all states
+    background-color: $euiColorEmptyShade  !important; // Override all states
```

Comment:
nit: I believe this was unintentional?

Reply:
so odd... I must have hit space or something... will fix
```suggestion
    background-color: $euiColorEmptyShade !important; // Override all states
```

Reply:
Looks like eslint is adding the extra space.  I'll ping #kibana-operations.